### PR TITLE
Ft comment crud 165412879

### PR DESCRIPTION
**Pull request by luc-tuyishime**
#### What does this PR do?
Allow user to comment on articles
#### Description of Task to be completed?
- Design a comment form where users will easily add, edit and delete comments on articles
#### How should this be manually tested?
- After cloning the repo, cd into it and type `npm test`.
- After cloning the repo, cd into it and type `npm start` and follow the instruction to open it in the browser.
#### Any background context you want to provide?
- To be able to add comments, you have to create an account, create articles and follow different users.
#### What are the relevant pivotal tracker stories?
- [#165412879](https://www.pivotaltracker.com/story/show/165412879)
#### Screenshots (if appropriate)
image

### CODE REVIEW

**Review by MBenedicte — APPROVED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```
Comment (e-liyai) [inline .vscode/settings.json]: why is this file added

File: src/__tests__/components/Articles/__snapshots__/ProgressBar.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (e-liyai) [inline src/__tests__/components/Articles/__snapshots__/ProgressBar.test.js.snap]: why do we need the snapshots?

File: .vscode/settings.json (hunk 1 above)
Comment (luc-tuyishime) [reply .vscode/settings.json]: I have not added this file but it's already removed.

File: src/__tests__/components/Articles/__snapshots__/ProgressBar.test.js.snap (hunk 2 above)
Comment (luc-tuyishime) [reply src/__tests__/components/Articles/__snapshots__/ProgressBar.test.js.snap]: honestly, we don't need it  and it's already removed

File: src/__tests__/components/comments/__snapshots__/commentForm.test.js.snap [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (e-liyai) [inline src/__tests__/components/comments/__snapshots__/commentForm.test.js.snap]: remove this also

Comment (luc-tuyishime) [reply src/__tests__/components/comments/__snapshots__/commentForm.test.js.snap]: Already removed
